---
layout: page
title: Ongoing and Past Projects
description: Yanhong's projects
---
Ongoing Projects

<div class="navbar">
    <div class="navbar-inner">
        <ul class="nav">
            <li><a href="#book">book</a></li>
            <li><a href="#articles">articles</a></li>
            <li><a href="#editorials">editorials</a></li>
            <li><a href="#letters">letters</a></li>
            <li><a href="#chapters">chapters</a></li>
            <li><a href="#techreports">tech reports</a></li>
            <li><a href="#thesis">dissertation</a></li>
        </ul>
    </div>
</div>

<!--
### <a name="book"></a>book

**Broman KW**, Sen &#346; (2009) A Guide to QTL Mapping with R/qtl.  Springer, New York
[![Online complements](icons16/html-icon.png)](http://www.rqtl.org/book)
[![Amazon](icons16/amazon-icon.png)](https://www.amazon.com/gp/product/0387921249?ie=UTF8&tag=7210-20)
[![Springer](icons16/springer-icon.png)](http://www.springer.com/978-0-387-92124-2)
[![R/qtl](icons16/R-icon.png)](http://www.rqtl.org)

### <a name="articles"></a>journal articles

#### in preparation

Boehm FJ, Chesler EJ, Yandell BS, **Broman KW** (2019)
Testing pleiotropy vs. separate QTL in multiparental populations.
[bioRxiv](https://www.biorxiv.org/) doi:[10.1101/550939](https://doi.org/10.1101/550939)
[![pdf](icons16/pdf-icon.png)](https://www.biostat.wisc.edu/~kbroman/publications/qtl2pleio.pdf)
[![GitHub](icons16/github-icon.png)](https://github.com/fboehm/qtl2pleio-manuscript)
[![qtl2pleio software](icons16/R-icon.png)](https://github.com/fboehm/qtl2pleio)
[![doi](icons16/doi-icon.png)](https://doi.org/10.1101/550939)

**Broman KW**, Gatti DM, Svenson KL, Sen &#346;, Churchill GA (2019)
Cleaning genotype data from Diversity Outbred mice.
[bioRxiv](https://www.biorxiv.org/) doi:[10.1101/518308](https://doi.org/10.1101/518308)
[![pdf](icons16/pdf-icon.png)](https://www.biostat.wisc.edu/~kbroman/publications/mpp_diag.pdf)
[![GitHub](icons16/github-icon.png)](https://github.com/kbroman/Paper_MPPdiag)
[![doi](icons16/doi-icon.png)](https://doi.org/10.1101/518308)

Lobo AK, Traeger LL, Keller MP, Attie AD, Rey FE, **Broman KW** (2019)
Identification of sample mix-ups and mixtures in microbiome data in
Diversity Outbred mice.
[bioRxiv](https://www.biorxiv.org/) doi:[10.1101/529040](https://doi.org/10.1101/529040)
[![pdf](icons16/pdf-icon.png)](https://www.biostat.wisc.edu/~kbroman/publications/mb_mixups.pdf)
[![GitHub](icons16/github-icon.png)](https://github.com/kbroman/Paper_MBmixups)
[![R/mbmixture software](icons16/R-icon.png)](https://github.com/kbroman/mbmixture)
[![doi](icons16/doi-icon.png)](https://doi.org/10.1101/529040)


#### 2019

**Broman KW**, Gatti DM, Simecek P, Furlotte NA, Prins P, Sen &#346;,
Yandell BS, Churchill GA (2019)
R/qtl2: software for mapping quantitative trait loci with
high-dimensional data and multi-parent populations.
[Genetics](http://genetics.org) 211:495-502
[![PubMed](icons16/pubmed-icon.png)](https://www.ncbi.nlm.nih.gov/pubmed/30591514)
[![pdf](icons16/pdf-icon.png)](https://www.biostat.wisc.edu/~kbroman/publications/rqtl2_paper.pdf)
[![GitHub](icons16/github-icon.png)](https://github.com/kbroman/Paper_Rqtl2)
[![R/qtl2 software](icons16/R-icon.png)](https://kbroman.org/qtl2)
[![doi](icons16/doi-icon.png)](https://doi.org/10.1534/genetics.118.301595)


#### 2018

Palus M, Sohrabi Y, **Broman KW**, Strnad H, Šíma M, Růžek D, Volkova
V, Slapničková M, Vojtíšková J, Mrázková L, Salát J, Lipoldová M
(2018) A novel locus on mouse chromosome 7 that influences survival
after infection with tick-borne encephalitis virus. [BMC
Neurosci](https://bmcneurosci.biomedcentral.com/) 19:39
[![PubMed](icons16/pubmed-icon.png)](https://www.ncbi.nlm.nih.gov/pubmed/29976152)
[![pdf](icons16/pdf-icon.png)](https://www.biostat.wisc.edu/~kbroman/publications/lipoldova2018.pdf)
[![doi](icons16/doi-icon.png)](https://doi.org/10.1186/s12868-018-0438-8)
-->
<!-- * 152 * -->
<!-- https://doi.org/10.1186/s12868-018-0438-8 -->



